tests/e2e/profile-happy-path.test.ts

```diff
@@ -0,0 +1,143 @@
+import {
+  closeBrowser,
+  openBrowser,
+  text,
+  click,
+  textBox,
+  into,
+  write,
+  clear,
+  link,
+  below,
+  waitFor,
+  currentURL,
+  screenshot,
+} from "taiko";
+
+import { authenticateToConnect } from "./utils/authenticate-to-connect";
+import { configVariables } from "@src/configs/config-variables";
+
+describe("Profile Happy path", () => {
+  jest.setTimeout(70000);
+
+  beforeAll(async () => {
+    await openBrowser({
+      args: [
+        "--window-size=1440,1000",
+        "--no-sandbox",
+        "--start-maximized",
+        "--disable-dev-shm",
+      ],
+      headless: true,
+      observe: false,
+      observeTime: 2000,
+    });
+  });
+
+  afterAll(async () => {
+    await closeBrowser();
+  });
+
+  test("it should do Profile and Addresses flows happy path", async (done) => {
+    expect.assertions(19);
+
+    try {
+      await authenticateToConnect();
+
+      const baseURL = (
+        process.env.CONNECT_TEST_ACCOUNT_URL ||
+        configVariables.connectAccountURL
+      ).replace(/\/?$/, "");
+
+      expect(await text("Personal Information").exists()).toBeTruthy();
+      await click("Personal Information");
+
+      // Update user profile flow
+      expect(
+        await link("Update your personal information").exists(),
+      ).toBeTruthy();
+      await click("Update your personal information");
+      expect(await text("Profile | edit").exists()).toBeTruthy();
+
+      await clear(textBox("Username"));
+      await write(
+        "Superman",
+        into(textBox({ placeholder: "Enter your username" })),
+      );
+
+      await click("Update my information");
+      expect(await text("Your profile has been updated").exists()).toBeTruthy();
+
+      // Add address flow
+      expect(await link("Add new address").exists()).toBeTruthy();
+      await click(link("Add new address"));
+      expect(await text("Address | new").exists()).toBeTruthy();
+
+      await write(
+        "Metropolis",
+        into(textBox({ placeholder: "Enter your locality" })),
+      );
+      await write(
+        "59000",
+        into(textBox({ placeholder: "Enter your postal code" })),
+      );
+      await write(
+        "United States of America",
+        into(textBox({ placeholder: "Enter your country" })),
+      );
+
+      await click("Add address");
+      await waitFor(250);
+      expect(await currentURL()).toEqual(`${baseURL}/account/profile`);
```

Comment:
I had to replace the alert message existence check by the check of the current URL here, because Taiko wasn't able to find the alert message (once again, this is an issue with Taiko speed while not on headless mode, because when you run the test file with headless mode, it works as expected.) I didn't find any other working workaround, and since when the flow (here adding an address) is successful, we are redirected to a precise URL, I thought it would count as a valid check.